README.md

```diff
@@ -49,6 +49,34 @@ npx hardhat deploy  --network localhost
 
 Hardhat re-uses old deployments, to force re-deploy add the `--reset` flag above
 
+## ZKLLVM output check
+
+Place folder with ZKLLVM output to `contracts/zkllvm` directory.
+
+Deploy contracts
+```
+npx hardhat deploy
+```
+
+Verify all folders from `contracts/zkllvm` director
+```
+npx hardhat verify
+```
+
+Verify one folder from `contracts/zkllvm` directory
+```
+npx hardhat verify-one folder-name
+```
+
+ZKLLVM output folder format
+```
+* proof.bin -- placeholder proof file
```

Comment:
Also, proof can be get from Proof Market as well, not only from ZKLLVM.
@hgedia what do you think, how to better put this fact here?

Reply:
I feel we might end up confusing users if we suggest this. The argument is if we plan to have a single repository for each circuit proof. (ex. Mina) - I feel users should use the driving repository to test the proof (ex : mina-state-proof). 

I feel this flow should be only used by circuit developers who are transpiling using zkllvm in the development workflow. Happy to discuss further.

Reply:
Well, fair enough